noir-projects/aztec-nr/aztec/src/macros/functions/utils.nr

```diff
@@ -92,7 +102,7 @@ pub(crate) comptime fn transform_private(f: FunctionDefinition) {
 
     // Inject the authwit check if the function is marked with #[authorize_once].
     let authorize_once_check = if fn_has_authorize_once(f) {
-        create_authorize_once_check(f)
+        create_authorize_once_check(f, true)
```

Comment:
This boolean makes me a bit sad, but at least in the IDE the editor will display it as `is_private: true`, plus we'll change the `auth_once` macro anyway, plus this is all very internal, plus there are no enums in Noir.

But I'm still sad.